File: src/SOP8/Utility/Response/PagingMessageResult.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Response
{
    public class PagingMessageResult : MessageResult
    {
        private int m_nTotalCount = 0;

        // 전체 아이템 개수
        public int TotalCount
        {
            get { return m_nTotalCount; }
            set { m_nTotalCount = value; }
        }

        public PagingMessageResult()
            : base()
        {
        }

        public PagingMessageResult(bool success, string message)
            : base(success, message)
        {
        }
    }
}

File: src/SOP8/광양/EgovLogin/base-utility/src/main/java/egovframework/base/response/PagingMessageResult.java
package egovframework.base.response;

import egovframework.base.response.MessageResult;

public class PagingMessageResult extends MessageResult {
    private int totalCount = 0;

    // 전체 아이템 개수 Getter / Setter
    public int getTotalCount() {
        return totalCount;
    }

    public void setTotalCount(int totalCount) {
        this.totalCount = totalCount;
    }

    // 기본 생성자
    public PagingMessageResult() {
        super();
    }

    // 매개변수가 있는 생성자
    public PagingMessageResult(boolean success, String message) {
        super(success, message);
    }
}